src/sr_state.erl

```diff
@@ -20,8 +20,14 @@
 %%% Types
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+-type options() ::
+  #{ path => string()
+   , model => atom()
```

Comment:
Yeah, but if you do so, make sure to put the right `min_otp_version` (or whatever the parameter name is) in rebar.config